host-configs/bgqos_0/clang_4_0_0.cmake

```diff
@@ -4,9 +4,6 @@
 #
 # SPDX-License-Identifier: (MIT)
 ##############################################################################
-set(CMAKE_CXX_COMPILER "/usr/apps/gnu/clang/r284961-stable/bin/bgclang++11" CACHE PATH "")
```

Comment:
These changes will break our current Bamboo builds

Reply:
Can I test bamboo builds on CZ ?

Reply:
Are bamboo tests using the same host-configs file as CZ at some point, or are they completely separated ?

Reply:
Nope, but look at the `scripts/bamboo/build_and_test.sh` and you will see the problem. We are using the project host-config to select compilers.

Reply:
This should now be corrected by the third commit.